find.go

```diff
@@ -53,6 +53,106 @@ func (s *server) findMultihashSubtree(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *server) findMetadata(w http.ResponseWriter, r *http.Request) {
+	discardBody(r)
+	if r.Method != http.MethodGet {
+		http.Error(w, "", http.StatusNotFound)
+		return
+	}
+
+	rcode, resp := s.doFindMetadata(r.Context(), r.Method, findMethodOrig, r.URL)
```

Comment:
Some context about `findMethodOrig`: the metrics were added to compare performance of vanilla http find API vs reframe because of timeouts and latency issues we experienced in the past. We should not be passing in `findMethodOrig` here because it will report the wrong latency values and could cause confusion when looking at metrics. See my comment on metrics.